code/modules/hydroponics/grown.dm

```diff
@@ -63,7 +74,7 @@
 				eat_time = eat_time,\
 				tastes = tastes,\
 				eatverbs = eatverbs,\
-				bite_consumption = bite_consumption_mod ? 1 + round(max_volume / bite_consumption_mod) : bite_consumption,\
```

Comment:
This was the original bug - notice `bite_consumption = max_volume / bite_consumption_mod` instead of `reagents.total_volume / bite_consumption_mod` like the comment above implies